<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportForm</class>
 <widget class="QWidget" name="ImportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <widget class="QPushButton" name="refreshCameraButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Scene Cameras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bakeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Bake Keys</string>
   </property>
  </widget>
  <widget class="QLabel" name="cameraListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scene Camera List</string>
   </property>
  </widget>
  <widget class="QLabel" name="fpsLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </widget>
  <widget class="QLabel" name="importLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Camera File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectedFileLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a .camout file!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="fpsTextEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="cameraListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="autoRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto FPS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="customRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom FPS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
